test/views/ViewTestHelper.scala

```diff
@@ -37,6 +37,24 @@ trait ViewTestHelper extends SpecBase {
     view.title() mustBe s"${messages(titleMsgKey)} - ${messages("service.name")} - GOV.UK"
 
   def shouldContainBackLinkUrl(view: Document,
-                               url: String): Assertion =
-    view.html().contains(url) mustBe true
+                               url: String): Assertion = view.html().contains(url) mustBe true
+
+  def shouldContainTheElement(view: Document,
+                              id: Option[String] = None,
+                              classes: Option[String] = None): Assertion = {
+    val isElementIdPresent = id.fold(false)(id => view.select(s"#$id").size() >= 0)
+    val isElementClassPresent = classes.fold(false)(classes => view.select(s".$classes").size() >= 0)
+
+    assert(isElementIdPresent || isElementClassPresent)
+  }
+
+  def shouldNotContainTheElement(view: Document,
```

Comment:
Here you are checking if the size is exactly `0`. This logic is fine, but the variable name `isElementIdPresent`can be confusing since it implies the element is present, when in fact the check is for the element not being present. Maybe rename to `isElementIdAbsent` and `isElementClassAbsent`.

Reply:
Done